### PR TITLE
[DOES NOT ACTUALLY WORK] Fix crashes on Chrome 43

### PR DESCRIPTION
**Pull request by paulcbetts**
This is my terrible attempt to fix crashes because of V8 handle clearing in Chrome 43 /cc @zcbenz 

## TODO:

- [ ] Actually get this to work
- [ ] Code Review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,6 @@
     "jasmine-focused": "1.x"
   },
   "dependencies": {
-    "nan": "https://atom.io/download/atom-shell/nan-1.6.1.tgz"
+    "nan": "*"
```
Comment (zcbenz) [inline package.json]: :heart: this.

File: src/main.cc [hunk 2]
```diff
@@ -75,17 +68,16 @@ class Spellchecker : public ObjectWrap {
     std::vector<std::string> dictionaries =
       that->impl->GetAvailableDictionaries(path);
 
-    Local<Array> result = NanNew<Array>(dictionaries.size());
+    Handle<Array> result = NanNew<Array>(dictionaries.size());
```
Comment (zcbenz) [inline src/main.cc]: Recent V8 encourages to use `Local` instead of `Handle`.

**Comment by paulcbetts** [comment]
@zcbenz Even with these fixes, spellchecker is still crashing - still trying to track it down

**Comment by paulcbetts** [comment]
@zcbenz So, I've been hacking on this more today trying to isolate what causes the crash, and it seems to come down to this line: https://github.com/atom/node-spellchecker/pull/16/files#diff-ddaa7d6e643bc0d374add600b02445beR43

Unboxing the string will cause the renderer to crash in an unrelated part of the code, even if you don't actually _do_ anything with the string. Does this ring any bells? It seems like it's some weird memory corruption but it's super consistent.

**Comment by zcbenz** [comment]
@paulcbetts Unfortunately I'm unable to reproduce the crash on both Windows and Mac with electron v0.28.1, I tested with a simple page like this:

```html
<body>
<input type="text" name="some_name" value="">

<script type="text/javascript" charset="utf-8">
  require('remote').getCurrentWindow().show();
  require('web-frame').setSpellCheckProvider("en-US", true, {
    spellCheck: function(text) {
      return !(require('spellchecker').isMisspelled(text));
    }
  });
</script>
</body>
```

And the module is installed with [npm](https://github.com/atom/electron/blob/master/docs/tutorial/using-native-node-modules.md#the-npm-way). Can you remove `node_modules` and rebuild again?

File: package.json (hunk 1 above)
Comment (steelbrain) [inline package.json]: @paulcbetts any reasons to not use `latest` instead of `*`

**Comment by paulcbetts** [comment]
> Can you remove node_modules and rebuild again?

I've rebuilt it a ton of times, it's a consistent repro for me on multiple operating systems :(  Lemme get you a crash dump / call stack

**Comment by paulcbetts** [comment]
Ok! I managed to track this down, this is _not_ a bug in node-spellchecker, due to what can be described as a Comedy of Errors, we were attempting to set the spellchecker in the callback for the spellchecker

**Pull request closed without merging**